plugin-server/tests/main/teardown.test.ts

```diff
@@ -1,11 +1,13 @@
+import ClickHouse from '@posthog/clickhouse'
 import { PluginEvent } from '@posthog/plugin-scaffold'
 
+import { waitForExpect } from '../../functional_tests/expectations'
```

Comment:
This was already imported like this in `plugin-server/tests/main/ingestion-queues/session-recording/session-recordings-consumer.test.ts` so I don't feel _too_ bad about it, but it seems like this helper could use a different home? (Or is there some generic retry that we could use instead?)